ex2/ex2.c

```diff
@@ -8,7 +8,9 @@
 
 int main(void)
 {
-    // Your code here 
-    
+    FILE *fp;
```

Comment:
You're writing to the file, but the point of the exercise is to do a fork and write to it from two processes. If you're doing it correctly, you should see two different statements in your text file.